lib/components/option-menus/index.js

```diff
@@ -84,6 +85,12 @@ module.exports = () => {
       e.preventDefault();
       target.click();
     }
+
+    if (which === 9) {
```

Comment:
these 2 if blocks could be combined into just 1.  In addition, I think we could turn `menuOpen` into a boolean - `const menuOpen = trigger.getAttribute('aria-expanded') === 'true'`

so the block would look like:
```js
if (which === 9 && menuOpen) {
  trigger.click();
}
```